src/services/db_project_handler.py

```diff
@@ -276,3 +279,25 @@ def delete_logo(self,
             raise ex
         else:
             db.commit()
+    
+
+    def email_invite(self,
+                     project_id: int,
+                     invite_sender_username: str,
+                     invite_receiver: str,
+                     email: str,
+                     db: Session):
+        proj = db.get(Projects, project_id)
+        owner = db.get(Users, invite_sender_username)
+        token_claims = {"sub": invite_receiver, "project": project_id}
+        join_token = create_join_token(to_encode=token_claims)
+        text = f"""Hello,
+        {owner.name} invited you to join the project '{proj.name}'.
+        To accept the invite go to: http://<url-where-app-is-running>/join?project_id={project_id}&join_token={join_token}
+        This invite is valid for 3 days. This is an automatic email, do not reply to this address. For additional info reply to {owner.email}"""
+        message_id = SESService().send_email_via_ses(text=text, to_address=email)
+        return SentEmailProjectInvite(aws_message_id=message_id,

```

Comment:
In general logic is okay
Buuut there is one nuance. You don't stick to SOLID principles (S - single-responsibility)
This class is intended to work with Database (the name is DBHandler). Yes, you have database operations in this method. But additionally, you integrated with SES here. Thus, in addition, this class started to work with AWS service

It's cleaner to separate two functionalities: get additional info from DB and send email via SES
For example, 
DBHandler.get_email_info -> project, owner instances
InvitationHandler (new class, __init__(db_handler: DBHandler, ses_service: SESService)). send_invitation(project, owner) 
or at least a separate function, if you don't want to create a class that has only one method